inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/view/InAppMessageBaseView.kt

```diff
@@ -58,16 +63,15 @@ internal open class InAppMessageBaseView(context: Context, attrs: AttributeSet?)
             this.messageBodyColor = Color.BLACK
             this.bgColor = Color.WHITE
         }
-
+        this.imageWidth = getDisplayWidth(context)
```

Comment:
Why use display width/height for image size? Why not use aspect ratio instead?

Reply:
Picasso will need both width and height to resize the image into the view

Reply:
Please add code comment